src/main/java/com/umc/networkingService/domain/university/service/UniversityServiceImpl.java

```diff
@@ -1,20 +1,170 @@
+                        ).build();
+    }
+
+    @Transactional(readOnly = true)    //우리 대학교 마스코트 조회
+    public UniversityResponse.joinUniversityMascot joinUniversityMascot(Member member){
+
+        UniversityResponse.joinUniversityMascot universityMascot
+                = UniversityConverter.toJoinUniversityMascot(member.getUniversity());
+
+        //지부 찾기
+        Branch branch = branchUniversityRepository.findByUniversityAndIsActive(member.getUniversity(),true)
+               .orElseThrow(() -> new RestApiException(ErrorCode.EMPTY_BRANCH)).getBranch();
+
+        List<University> universityRankList = universityRepository.findAllByOrderByTotalPointDesc(); //랭킹 순 정렬
+
+        return UniversityResponse.joinUniversityMascot.setRankAndBranch(
+                universityMascot
+                ,universityRankList.indexOf(member.getUniversity())+1
+                ,branch
+        );
+    }
+
+    @Transactional    //우리 대학교 마스코트 먹이주기  todo: 마스코드 레벨업, 마스코트 변경
+    public void feedUniversityMascot(Member member, PointType pointType){
+
+        if(member.getRemainPoint() < pointType.getPoint()){
+            throw new RestApiException(ErrorCode.NOT_ENOUGH_POINT);
+        }
+        
+        //포인트 차감
+        member.usePoint(pointType.getPoint());
+        //학교 포인트 증가
+        member.getUniversity().increasePoint(pointType.getPoint());
+
+        //포인트 히스토리 추가
+        memberPointRepository.save(
+                MemberPoint.builder()
+                        .member(member)
+                        .pointType(pointType)
+                        .university(member.getUniversity())
+                        .build()
+        );
+
+
+    }
+
+    @Transactional    //학교 생성
+    public UUID createUniversity(UniversityRequest.createUniversity request){
+        if(universityRepository.findByName(request.getUniversityName()).isPresent()){
+            throw new RestApiException(ErrorCode.DUPLICATE_UNIVERSITY_NAME);
+        }
+        universityRepository.save(University.builder()
+                .name(request.getUniversityName())
+                .universityLogo(uploadImage("university",request.getUniversityLogo()))
+                .semesterLogo(uploadImage("semester",request.getSemesterLogo()))
+                .build());
+        return universityRepository.findByName(request.getUniversityName()).get().getId();
+    }
+
+    @Transactional   //학교 삭제
+    public void deleteUniversity(UUID universityId){
+        University university = universityRepository.findById(universityId)
+                .orElseThrow(() -> new RestApiException(ErrorCode.EMPTY_UNIVERSITY));
+        university.delete();
+    }
+
+    @Transactional    //학교 정보 수정
+    public void patchUniversity(UniversityRequest.patchUniversity request){
+
+        University university;
+        university = universityRepository.findById(request.getUniversityId())
+                .orElseThrow(() -> new RestApiException(ErrorCode.EMPTY_UNIVERSITY));
```

Comment:
혹시 선언부를 따로 구분해 놓은 이유가 있을까요?? 아니라면, 아래처럼 구분하지 않는 것도 좋을거 같아요!!
```suggestion
        University university = universityRepository.findById(request.getUniversityId())
                .orElseThrow(() -> new RestApiException(ErrorCode.EMPTY_UNIVERSITY));
```